lib/Controller/SieveController.php

```diff
@@ -109,7 +113,13 @@ public function updateActiveScript(int $id, string $script): JSONResponse {
 		$sieve = $this->getClient($id);
 
 		$scriptName = $sieve->getActive() ?? 'nextcloud';
-		$sieve->installScript($scriptName, $script, true);
+
+		try {
+			$sieve->installScript($scriptName, $script, true);
+		} catch (ManagesieveException $e) {
+			$this->logger->error('Installing sieve script failed: ' . $e->getMessage(), ['app' => 'mail']);
```

Comment:
```suggestion
			$this->logger->error('Installing sieve script failed: ' . $e->getMessage(), ['app' => 'mail', 'exception' => $e]);
```

to log the trace

https://www.php-fig.org/psr/psr-3/#13-context